pkg/providers/vsphere/template.go

```diff
@@ -319,5 +345,17 @@ func buildTemplateMapMD(
 		values["bottlerocketBootstrapVersion"] = bundle.BottleRocketBootstrap.Bootstrap.Tag()
 	}
 
-	return values
+	return values, nil
+}
+
+func etcdMachineConfig(spec *cluster.Spec) *anywherev1.VSphereMachineConfig {
```

Comment:
Instead of defining a new method here, can we re-use this one? https://github.com/aws/eks-anywhere/blob/15bb295f98245fb418fcb2b086ca9733bf83e62d/pkg/providers/vsphere/spec.go#L27

Same for CP and worker node configs